features/brief_results_display.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Brief result display
+  In order to view relevant fields in search results
+  As a researcher
+  I want to see relevant fields in the brief search display.
+
+  
+  Scenario: See brief results display at the collection level 
+    Given I am on the brief results page
+    And I limit my search to "Archival Collection" under the "Format" category
```

Comment:
@eshadatta -- Looks like this one is failing too, but will pass if you change "Format" to "Level".